### PR TITLE
Web graphics sep

### PR DESCRIPTION
**Pull request by cshartsough**
Adds 5-6 new snow/freezing precip fields. Adds a fix for thickness overlay.  New conversion test.

pylint and pytest completed.

Will add figures next.

### CODE REVIEW

**Comment by cshartsough** [comment]
[web-graphics_PR-54.pptx](https://github.com/NOAA-GSD/adb_graphics/files/5222005/web-graphics_PR-54.pptx)

**Review by christinaholtNOAA — CHANGES_REQUESTED** [review]
The graphics look good. I like that you changed 1hr acc snow colors to be consistent with the other products. :)

A bit worried (currently) about the thickness fix. Let me check that out and get back to you.

File: adb_graphics/default_specs.yml [hunk 1]
```diff
@@ -318,7 +356,7 @@ G124bt: # GOES-E Brightness temperature, infrared (Ch 4)
     unit: ch 4
 gh: # Geopotential height
   500mb: &ua_gh
-    clevs: !!python/object/apply:numpy.arange [6, 990, 6]
+    clevs: !!python/object/apply:numpy.arange [550, 600, 5]
```
[the remark this answers is not in the document]
Comment (cshartsough) [reply adb_graphics/default_specs.yml:359]: It's no problem to set the interval to 6, I was just debugging last night with different clevs.

I don't know what the original clevs range is for, unless we only expect to use the field as overlays and don't care how big a set of levels it generates.  I can put it back to the original values.

[the remark this answers is not in the document]
Comment (christinaholtNOAA) [reply adb_graphics/default_specs.yml:359]: Okay! I found the bug I introduced that causes this problem. In datahandler/grib.py, near line 158, it should read:

```lev_val, _ = self.numeric_level(level=level)```

You want to pop that in and remove all the changes to the default_specs.yml for these two gh sections to see what happens?

[the remark this answers is not in the document]
Comment (cshartsough) [reply adb_graphics/default_specs.yml:359]: Yes, that seems to work.  Thanks.

File: adb_graphics/default_specs.yml [hunk 2]
```diff
@@ -75,25 +85,53 @@
...
+    cmap: gist_ncar
+    colors: accumulated_precip_colors
+    ncl_name: ASNOW_P8_L1_GLC0_acc
+    ticks: 0
+    title: Run Total Accumulated Snow Depth - var dens
+    transform: conversions.m_to_in
+    unit: in
+acsnw: # Run Total Accumulated Snow Using 10:1 Ratio
+  sfc:
+    clevs: [0.01, 0.1, 1, 2, 3, 4, 6, 8, 10, 12, 18, 24]
```
[the remark this answers is not in the document]
Comment (cshartsough) [reply adb_graphics/default_specs.yml:124]: I anchored acfrzr and acfrozr.  I didn't anchor acpcp, but did merge color tables for accumulated_precip_hourly, rh_colors, and hlcy_colors, into rainbow12_colors.

**Review by christinaholtNOAA — APPROVED** [review]
Looks great! Thanks for the changes.

**Review by Brian-Jamison — APPROVED** [review]
All looks good!  Thanks!

**Pull request merged**